src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewIconWithLinkTextCell.swift

```diff
@@ -0,0 +1,96 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+class DynamicTableViewIconWithLinkTextCell: UITableViewCell {
+
+	// MARK: - Init
+
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+		super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+		backgroundColor = .enaColor(for: .background)
+
+		stackView.axis = .horizontal
+		stackView.spacing = 16
+
+		contentView.addSubview(stackView)
+		stackView.translatesAutoresizingMaskIntoConstraints = false
+
+		imageViewWidthConstraint = iconImageView.widthAnchor.constraint(equalToConstant: 32)
+		imageViewWidthConstraint?.isActive = true
+
+		stackView.addArrangedSubview(iconImageView)
+
+		contentTextView.setContentCompressionResistancePriority(.required, for: .vertical)
+		stackView.addArrangedSubview(contentTextView)
+
+		NSLayoutConstraint.activate([
+			stackView.leadingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.leadingAnchor),
+			stackView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 12),
+			stackView.trailingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.trailingAnchor),
+			stackView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -12),
+			iconImageView.widthAnchor.constraint(equalTo: iconImageView.heightAnchor)
+		])
+	}
+
+	// MARK: - Internal
+
+	// swiftlint:disable function_parameter_count
+	func configure(
+		image: UIImage?,
+		imageAlignment: DynamicCell.ImageAlignment = .left,
+		text: String,
+		links: [ENALinkedTextView.Link],
+		customTintColor: UIColor?,
+		style: ENAFont,
+		iconWidth: CGFloat,
+		selectionStyle: UITableViewCell.SelectionStyle,
+		alignment: UIStackView.Alignment
+	) {
+		stackView.alignment = alignment
+
+		iconImageView.tintColor = customTintColor ?? tintColor
+		iconImageView.image = image
+		iconImageView.isHidden = image == nil
+		
+		// swap label and image so the image is set to the right. Do this every time to get a clean reusable state.
```

Comment:
Why not handle this in `prepareForReuse` ?

Reply:
I don't understand this. Why should it be in prepareForReuse?

Reply:
"Do this every time to get a clean reusable state." Feels like it belongs in prepareForReuse

Reply:
I copied this code from the other component. I think the comment is misleading. It means that we need to check both cases and adjust the UI, because it might get reused.

Regarding prepareForReuse:
From the docs:
"To avoid potential performance issues, you should only reset attributes of the cell that are not related to content, for example, alpha, editing, and selection state. The table view's delegate in tableView(_:cellForRowAt:) should always reset all content when reusing a cell."

As far as I understand, our code is OK.